### DISCUSSION TITLE
有没有便宜点的云用啊

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#844] [Q&A]
最好是腾讯轻应用这种，带宽贼大的，4G 一年 100 以内都能接受

**Comment by Arondight** [comment]
到期之后准备先用树莓派3b熬着，熬到出头那一天

**Comment by Xm798** [comment]
双11可能便宜些，我去年买的 2c4g8m 的腾讯云轻量三年只要 150，后来给免费升成了 4c4g8m，现在感觉普遍贵了，可能腾讯也觉得亏了（（
